change/@fluentui-react-native-adapters-2020-05-26-21-34-00-lehon-keyboardClick.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Why do we need to add change file to unrelated packages like this one? @JasonVMo